src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--start-day--sun calendar--month-length--30">
```

Comment:
The naming of the modifiers should be calendar--start-day-sun instead of calendar--start-day--sun. Apply this change to other start-day and month-length modifiers as well.